xml/book_poc_crowbar.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<!DOCTYPE section [
+ <!ENTITY % entities SYSTEM "entity-decl.ent"> %entities;
+]>
+<book xmlns="http://docbook.org/ns/docbook"
+      xmlns:xi="http://www.w3.org/2001/XInclude"
+      xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:lang="en"
+      xml:id="book.poc.crowbar">
+ <title>&crow; Proof of Concept Guide</title>
+ <info>
+ <xi:include href="copyright_suse_cloud.xml"/>
+  <abstract>
+   <para>
+     At the time of the &productnumber; release, this guide
+     is a proof of concept (POC) of a &productname; deployment.
+     This guide provides the basic steps for deployment
+     &productname; &productnumber; in a bare metal environment.
```

Comment:
for deployment
maybe deploying
or
deployment of

Reply:
I opted for _This guide describes a basic deployment of_ How does this sound?

Reply:
sounds fine.